pyproject.toml

```diff
@@ -1,59 +1,68 @@
-[tool.poetry]
+[project]
 name = "poetry"
 version = "2.0.0"
 description = "Python dependency management and packaging made easy."
-authors = ["Sébastien Eustace <sebastien@eustace.io>"]
+requires-python = ">=3.9,<4.0"
+dependencies = [
+    "poetry-core (==2.0.0)",
+    "build (>=1.2.1,<2.0.0)",
+    "cachecontrol[filecache] (>=0.14.0,<0.15.0)",
+    "cleo (>=2.1.0,<3.0.0)",
+    "dulwich (>=0.22.6,<0.23.0)",
+    "fastjsonschema (>=2.18.0,<3.0.0)",
+    "importlib-metadata (>=4.4) ; python_version < '3.10'",
+    "installer (>=0.7.0,<0.8.0)",
+    "keyring (>=25.1.0,<26.0.0)",
+        # packaging uses calver, so version is unclamped
```

Comment:
Indent to deep

Reply:
Shouldn't pre-commit have caught this?

Reply:
`check-toml` only validates the syntax, nothing else. An indented comment is still a valid TOML.